src/plugins/email/README.md

```diff
@@ -0,0 +1,56 @@
+- infos = Information about the email plugin is in keys below
+- infos/author = Andreas Kraschitzer <a-kraschitzer>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = check
+- infos/recommends =
+- infos/placements = presetstorage
+- infos/status = maintained unittest nodep
```

Comment:
If the plugin is experimental as suggested by the CMake component, you can denote it here.
Also, `nodep` somewhat indicates it is pure C99, but the plugin needs `POSIX` for the regexes afaik?
Add maintained only if you intend to maintain it.
```suggestion
- infos/status = maintained unittest nodep experimental
```